modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro

```diff
@@ -502,4 +502,17 @@ definition {
 		AntCommands.runCommand("build-test.xml", "unzip-temp-file -DfileName=Export.zip");
 	}
 
+	@summary = "Default summary"
+	macro unzipDownloadFileInTempFolder() {
```

Comment:
This only works when there is one zip file in temp folder, right? Wondering why use command --output is not working? :thinking:

Reply:
> This only works when there is one zip file in temp folder, right?

I'd say yes, but for this method, it can list an array list for all files in the temp folder, if you need to select the other file I think you can modify this method